sklearn/tree/_export.py

```diff
@@ -683,18 +687,20 @@ def export(self, decision_tree, ax=None):
     def recurse(self, node, tree, ax, max_x, max_y, depth=0):
         import matplotlib.pyplot as plt
 
-        kwargs = dict(
-            bbox=self.bbox_args.copy(),
+        # kwargs for annotations without a bounding box
+        non_box_kwargs = dict(
             ha="center",
             va="center",
             zorder=100 - 10 * depth,
             xycoords="axes fraction",
             arrowprops=self.arrow_args.copy(),
```

Comment:
I think `non_box_kwargs` should not have `arrowprops`.

Reply:
Done. 029e8c87b